packages/flutter_tools/test/general.shard/intellij/intellij_validator_test.dart

```diff
@@ -11,6 +11,8 @@ import 'package:flutter_tools/src/convert.dart';
 import 'package:flutter_tools/src/doctor_validator.dart';
 import 'package:flutter_tools/src/intellij/intellij_validator.dart';
 import 'package:flutter_tools/src/ios/plist_parser.dart';
+import 'package:path/path.dart';
```

Comment:
That's a good idea. I think the safe way to do this though would be to actually parse our code with `package:analyzer`, rather than the regex based matching we're doing now.

Instead, I just exported the `Context` interface from test/src/common.dart.